pilot_data_analysis.Rmd

```diff
@@ -219,76 +220,65 @@ ggplot(ms, aes(x = t, y = looking, col = region)) +
-
-Note that this is done ad-hoc for a number of participants. TODO - cleanup!
-
-```{r}
-df_t_first_aoi <- d %>% split(.$subid, .$trial_num) %>% 
-  map_df(t_first_aoi) %>% 
+```{r, eval = FALSE}
+#First, create several variables that indicate first_look_time in each trial for each subject, recode aoi into dummy variable and combine string 
+df_t_first_aoi <- d %>% 
+  filter(., experiment_num == "pilot_1a",
+         t >= -4000, t <= 120, 
+         aoi %in% c("target", "distractor")) %>% 
   mutate(aoi_dummy = case_when(aoi == "target" ~ 1, aoi == "distractor" ~ 0),
-         subid_trial = paste0(subid, trial_num)) 
-
-df_t_first_aoi <- df_t_first_aoi %>% 
-                  group_by(subid, trial_num) %>% 
-                  mutate(diff = aoi_dummy - lag(aoi_dummy))
-
-head_4 <- function(df){df %>% slice(1:4)}
-
-df_t_4 <- df_t_first_aoi %>% split(.$subid, .$trial_num) %>%
-                  map_df(head_4) %>%
-                  filter(diff != 0) %>% 
-                  mutate(subid_trial = paste0(subid, trial_num))
-
-unique(df_t_4$subid_trial) 
-
-df_t_change_kids <- df_t_first_aoi %>% 
-                    mutate(subid_trial = paste0(subid, trial_num)) %>% 
-                    filter(subid_trial %in% df_t_4$subid_trial) %>% 
-                    select(subid_trial, subid, trial_num, t, first_look_time, aoi, aoi_dummy)
-
-df_t_first_aoi <- df_t_first_aoi %>% 
-  select(subid_trial, subid, lab, stimulus, age, trial_num, t, first_look_time, aoi, aoi_dummy) %>% 
-  filter(t == first_look_time) %>% 
-  mutate(first_look_location = case_when(subid_trial == "lmu_child044" ~ 1,
-                                       subid_trial == "lmu_child111" ~ 0,
-                                       subid_trial == "lmu_child115" ~ 0,
-                                       subid_trial == "lmu_child121" ~ 1,
-                                       subid_trial == "mb2_p123" ~ 1,
-                                       subid_trial == "mb2_p164" ~ 1,
-                                       subid_trial == "mb2_p37" ~ 0,
-                                       subid_trial == "MB2_P46" ~ 0,
-                                       subid_trial == "MB2_P56" ~ 0,
-                                       subid_trial == "MB2-108" ~ 0,
-                                       subid_trial == "MB2-117" ~ 1,
-                                       subid_trial == "MB2-124" ~ 1,
-                                       subid_trial == "MB2-126" ~ 1,
-                                       subid_trial == "mb2p067" ~ 1,
-                                       subid_trial == "mb2p103" ~ 1,
-                                       subid_trial == "mb2p114" ~ NA_real_,
-                                       subid_trial == "mb2p126" ~ 0,
-                                       subid_trial == "mb2p127" ~ 1,
-                                       TRUE ~ aoi_dummy)) %>% 
-  filter(!is.na(first_look_location))
+         subid_trial = paste0(subid, sep = "_trial_", trial_num)) %>% 
```

Comment:
did you want to comment all this out?

Reply:
I will cut this out after Gal and I found a new way to compute the first look